tools/samples-build

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+#
+# This script builds the sample applications.
+
+set -e
+
+if [ -z $1 ] 
+then 
+    echo "No platform argument given, use all|linux|ios|android"
+    set $1 'linux'
+fi
+
+if [ ! "$1" = "linux" ] && [ ! "$1" = "ios" ] && [ ! "$1" = "android" ]
+then
+    echo "Wrong platform argument given, use all|linux|ios|android"
+    set $1 'linux'
+fi
+
+echo "Platform argument: "$1
+
+if [ "$1" = "linux" ] || [ "$1" = "all" ]
+then
+    pushd samples/cpp
+    echo "Building CPP sample app:"`pwd`
+    cmake . -DWALLET_CORE=../../
+    make
+    echo "Building CPP sample app: done"
+    popd
```

Comment:
can you create shell function for each platform? so we can do:

```bash
if [ "$1" = "all" ]; then
build_cpp()
build_ios()
build_android()
build_go()
fi
```

Reply:
Done